include/ck/tensor_operation/gpu/device/impl/device_gemm_dl.hpp

```diff
@@ -485,6 +492,50 @@ struct DeviceGemmDl : public DeviceGemm<ALayout,
 
     static bool IsSupportedArgument(const Argument& arg)
     {
+        // Make sure that the M, N, K dimensions before padding are divisible by respective vector
+        // lengths.
+        if constexpr(is_same<tensor_layout::gemm::RowMajor, ALayout>::value)
+        {
+            constexpr auto A_K_vec_length =
```

Comment:
Maybe use I1 instead of 1, I2 instead of 2? But if prefer, skip this comment.

Reply:
changed